tools/federation/src/main/java/org/eclipse/rdf4j/federated/evaluation/FederationEvalStrategy.java

```diff
@@ -147,6 +149,14 @@ public abstract class FederationEvalStrategy extends StrictEvaluationStrategy {
 
 	protected FederationContext federationContext;
 
+	/**
+	 * List of standard {@link QueryOptimizer}s applicable to federation
+	 */
+	protected static final List<QueryOptimizer> standardOptimizers = List.of(
```

Comment:
Does this need to be protected?

Reply:
no need in this case, adjusted (as only change in the pushed commit). Thanks for the review